### ISSUE TITLE
MapFromDb not working when using ReturnValues option

### ISSUE THREAD

**Issue by thomasdavis** [mhart/dynamo-table#10]
Hi there, great work on this lib!

```
    @_tables[table].update(id, values, {ReturnValues: 'ALL_NEW'}, (e,r) =>
      console.log(@_tables[table].mapFromDb(r))
      @_dynCb(e,r,table,id,'update',cb)
    )
```

When I don't exclude the `ReturnValues`, `r` comes back as a JSON object. When I use `ReturnVaues` it comes back as a Dynamo object, so I call mapFromDb on it but that throws an error

```
Error: Unknown DynamoDB type for "Attributes": {"_identifier":{"S":"asdas"},"_
.................
```

Curious to know your thoughts on the issue.

**Comment by mhart** [comment]
Ooh, it's been literally years since I've touched this library :grimacing: – it may very well be to do with updates to DynamoDB since – but I'll poke around and see what I can find.

**Comment by thomasdavis** [comment]
Oh don't stress about it, if I find it I will send through a PR. Thought you might know off the top of your head.

**Comment by mhart** [comment]
I believe this was resolved on Twitter? The resolution being to call `mapFromDb` on `r.Attributes`

**Comment by thomasdavis** [comment]
Thanks again!

On Tue, Feb 16, 2016 at 5:14 PM, Michael Hart notifications@github.com
wrote:

> Closed #10 https://github.com/mhart/dynamo-table/issues/10.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mhart/dynamo-table/issues/10#event-551579533.

## 

Thomas Davis
http://thomasdav.is

VP of Tech - Earbits - http://earbits.com
Co-founder - Cdnjs - http://cdnjs.com
Founder - Backbone Tutorials - http://backbonetutorials.com
